dashboard/test/controllers/api/v1/pd/applications_controller_test.rb

```diff
@@ -899,6 +899,7 @@ class ApplicationsControllerTest < ::ActionController::TestCase
         "Assigned Workshop",
         "Registered for workshop?",
         "Regional Partner",
+        "Link to Application",
```

Comment:
You've extended the tests to expect a column for the application link but there's nothing checking that the application link is well-formed (or even present, other than the header).  Is that sort of unit test something you'd consider overkill here, or was it just overlooked in this case?

Reply:
added url check in a test in this PR: https://github.com/code-dot-org/code-dot-org/pull/26725/files